### PR TITLE
Runing tests on IE backend 

### PR DESCRIPTION
**Pull request by l-bat**
### This pullrequest changes
The following test now use IE backend:    
Normalize layer
* Test_Torch_layers.net_normalize
* Test_Torch_nets.OpenFace_accuracy

Resize layer
* Test_Darknet_nets.YOLOv3
* Test_Darknet_layers.upsample
* Test_Torch_layers.upsampling_nearest  

LRN layer
* Test_Caffe_layers.LRN

<cut/>

### Performance test results 
Processor: Intel® Core™ i5-6500 CPU @ 3.20GHz × 4
Graphics: Intel® HD Graphics 530 (Skylake GT2)  

| Name of Test                            | Time before changes (ms) | Time after changes (ms) |  x-factor after vs before |
|-----------------------------------------|--------------------------|-------------------------|---------------------------|
| OpenFace::DNNTestNetwork::DLIE/CPU      | 18.298                   | 17.791                  | 0.97                      |
|  OpenFace::DNNTestNetwork::DLIE/OCL     | 8.375                    | 8.051                   | 0.96                      |
| OpenFace::DNNTestNetwork::DLIE/OCL_FP16 | 8.778                    | 7.491                   | 0.85                      |
| YOLOv3::DNNTestNetwork::DLIE/MYRIAD     | 1946.705                 | 1957.809                | 1.01                      |

```
force_builders=Custom
docker_image:Custom=ubuntu-openvino-2018r5:16.04
test_opencl:Custom=ON
buildworker:Custom=linux-2
#test_modules=dnn
allow_multiple_commits=1
```

### CODE REVIEW

**Comment by dkurt** [comment]
It looks like we can enable OCL_FP16 target as well:
https://github.com/opencv/opencv/blob/b068d26fad1de316de4d6357e38afcf2193f6e32/modules/dnn/test/test_torch_importer.cpp#L280-L281

As here:
https://github.com/opencv/opencv/blob/b068d26fad1de316de4d6357e38afcf2193f6e32/modules/dnn/test/test_backends.cpp#L226-L240

P.S. it looks like we need to refactor framework-specific tests and `test_backends.cpp` because there are duplicated runs.

File: modules/dnn/src/layers/normalize_bbox_layer.cpp [hunk 1]
```diff
@@ -65,16 +65,12 @@ class NormalizeBBoxLayerImpl CV_FINAL : public NormalizeBBoxLayer
     {
         if (backendId == DNN_BACKEND_INFERENCE_ENGINE)
         {
-            if (pnorm != 2)
+            if (!(pnorm == 2 || pnorm == 1)) {
```
Comment (dkurt) [inline modules/dnn/src/layers/normalize_bbox_layer.cpp]: `pnorm` is not used for IE layer creation so I think we still need to allow only `pnorm == 2` for now.

File: modules/dnn/test/test_torch_importer.cpp [hunk 2]
```diff
@@ -272,12 +272,12 @@ class Test_Torch_nets : public DNNTestLayer {};
...
 TEST_P(Test_Torch_nets, OpenFace_accuracy)
 {
-#if defined(INF_ENGINE_RELEASE) && (INF_ENGINE_RELEASE < 2018030000 || INF_ENGINE_RELEASE == 2018050000)
+#if defined(INF_ENGINE_RELEASE) && (INF_ENGINE_RELEASE < 2018030000)
     if (backend == DNN_BACKEND_INFERENCE_ENGINE && target == DNN_TARGET_MYRIAD)
         throw SkipTestException("");
 #endif
     checkBackend();
-    if (backend == DNN_BACKEND_INFERENCE_ENGINE && target == DNN_TARGET_OPENCL_FP16)
+    if (backend == DNN_BACKEND_INFERENCE_ENGINE && target == DNN_TARGET_OPENCL_FP16 && INF_ENGINE_RELEASE != 2018050000)
```
Comment (dkurt) [inline modules/dnn/test/test_torch_importer.cpp]: Let's keep this line unchanged. We test only the latest version (R5)

File: modules/dnn/src/layers/normalize_bbox_layer.cpp [hunk 3]
```diff
@@ -65,16 +65,12 @@ class NormalizeBBoxLayerImpl CV_FINAL : public NormalizeBBoxLayer
     {
         if (backendId == DNN_BACKEND_INFERENCE_ENGINE)
         {
-            if (pnorm != 2)
+            if (pnorm != 2) {
```
Comment (dkurt) [inline modules/dnn/src/layers/normalize_bbox_layer.cpp]: I think it's better to remove brackets.

**Comment by alalek** [comment]
Is it still **WIP**?

**Comment by dkurt** [comment]
@alalek, I removed it.

**Comment by l-bat** [comment]
Yes, I got result with `OMP_WAIT_POLICY=PASSIVE. ` Without this x-factor = 1.04

**Pull request merged**